app/services/doc_auth/socure/responses/docv_result_response.rb

```diff
@@ -105,6 +112,39 @@ def read_pii
           )
         end
 
+        def log_verification_request
+          user_id = nil
+          if !document_capture_session&.user.nil?
+            user_id = document_capture_session.user.uuid
+          end
+          @analytics.idv_socure_verification_data_requested(
+            success: doc_auth_success?,
+            errors: nil,
+            exception: nil,
+            async: false,
+            docv_transaction_token: document_capture_session&.socure_docv_transaction_token,
+            customer_profile: get_data(DATA_PATHS[:customer_profile]),
+            reference_id: get_data(DATA_PATHS[:reference_id]),
+            reason_codes: get_data(DATA_PATHS[:reason_codes]),
+            document_type: get_data(DATA_PATHS[:document_type]),
+            decision: get_data(DATA_PATHS[:decision]),
+            user_id: user_id,
+            state: get_data(DATA_PATHS[:state]),
```

Comment:
i recommend we  make state, issue_date,zip_code, birth_date, address_line2 methods for use by this method and read_pii to avoid divergence on the get_data calls performed in both methods